lib/routes/account.js

```diff
@@ -438,6 +438,7 @@ module.exports = (
         checkIsBlockForced()
           .then(() => customs.check(request, email, 'accountLogin'))
           .catch(extractUnblockCode)
+          .then(checkSecondaryEmail)
```

Comment:
@vbudhram most people will either have an account already or not at all. Does it make sense to move this check below the `readEmailRecord` check? If you find the email via `readEmailRecord `then you don't have to do `checkSecondaryEmail` it seems.